modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/form-view/EditFormView.es.js

```diff
@@ -61,20 +62,30 @@ const EditFormView = (props) => {
 	return (
 		<DndProvider backend={HTML5Backend}>
 			<FormViewContextProvider dataLayoutBuilder={dataLayoutBuilder}>
-				<FormViewControlMenu
-					backURL={backURL}
-					dataLayoutId={dataLayoutId}
-				/>
+				<div
+					className={classNames({
+						'publications-enabled': document.querySelector(
+							'.change-tracking-indicator'
+						),
+					})}
+				>
```

Comment:
@kevenleone @rebsilva Podemos adicionar essa div somente quando o publications for habilitado?

Reply:
É possível sim @interaminense , posso fazer este ajuste.
Que achas @rebsilva ?

Reply:
Acho que nesse caso aqui ficar melhor mesmo, porque se o publications não estiver ativado, a div é criada mas fica sem nada não é? Então é mais coerente apenas criar quando o publications estiver ativado

Reply:
Sim, e não impacta nos snapshots existentes também !!!